tests/all.sh

```diff
@@ -11,6 +11,13 @@ then
     exit 1;
 fi
 
+if [ "$1" != "" ]
```

Comment:
`FLICT` var could be set to `$1` in both cases.
Meaning`FLICT="$1"`should be enough here.